ts/util/table-builder.ts

```diff
@@ -72,7 +124,7 @@ export class TableBuilder<ParentType, ChildType> {
 
   public build() {
     const topControlsContainer = document.createElement("div");
-    topControlsContainer.className = "flex mt-4 items-center";
+    topControlsContainer.className = "flex mt-4 items-center space-x-2";
```

Comment:
Change `items-center` to `items-top` to improve display when there are enough filters to cause wrapping